protostar/commands/init/init_command.py

```diff
@@ -73,12 +75,16 @@ def init(
                 should_adapt_existing_project = self._requester.confirm(
                     "Your current directory may be a Cairo project.\n"
                     "Do you want to adapt current working directory "
-                    "as a project instead of creating a new project?."
+                    "as a project instead of creating a new project?"
                 )
 
         if should_adapt_existing_project:
             self._adapted_project_creator.run()
         else:
+            logger.info(
+                "Consider using OpenZeppelin Contracts Wizard to kickstart your project.\n"
+                "https://wizard.openzeppelin.com/cairo"
+            )
```

Comment:
We definiately shouldn't include such stuff in logs. We can consider adding it to the docs but I am not sure about this either